app/Espo/Services/Record.php

```diff
@@ -1369,6 +1369,10 @@ public function updateEntity($id, $data)
             $this->afterUpdateEntity($entity, $data);
 
             if (empty($this->getMemoryStorage()->get('importJobId'))) {
+                $entity = $this
+                    ->dispatchEvent('beforePrepareEntityForOutput', new Event(['id' => $id, 'data' => $data, 'entity' => $entity, 'service' => $this]))
```

Comment:
Bad name for the event. This name should be used inside the function, not outside.

Reply:
hope the new name is ok :)